Framework/Algorithms/src/CopySample.cpp

```diff
@@ -111,7 +115,8 @@ void CopySample::exec() {
     {
       for (uint16_t i = 0; i < outMDWS->getNumExperimentInfo(); i++)
         copyParameters(sample, outMDWS->getExperimentInfo(i)->mutableSample(), copyName, copyMaterial, copyEnvironment,
-                       copyShape, copyLattice, copyOrientation);
+                       copyShape, copyLattice, copyOrientation,
+                       outMDWS->getExperimentInfo(i)->run().getGoniometer().getR());
```

Comment:
You could do `auto ei = outMDWS->getExperimentInfo(i)` before the call to copyParameters just to make the argument list is a bit less verbose